### PR TITLE
Update selection on create; defaults for `selectedValues` and `selectedItems`.

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: test/multi.html [hunk 1]
```diff
@@ -85,9 +85,9 @@
       });
 
       test('has sane defaults', function() {
-        assert.equal(selector.selectedValues, undefined);
-        assert.equal(selector.selectedClass, 'iron-selected');
-        assert.equal(selector.items.length, 5);
+        assert.deepEqual(selected.selectedValues, []);
```
Comment (bicknellr) [inline test/multi.html]: s/selected/selector/g

Comment (valdrinkoshi) [inline test/multi.html]: 👍

**Comment by bicknellr** [comment]
@valdrinkoshi Ok, corrected the name. One more check?

**Comment by valdrinkoshi** [comment]
LGTM 👌

**Pull request merged**